src/js/components/Diagram/index.d.ts

```diff
@@ -17,10 +17,16 @@ export type DiagramAnimationType =
 
 export interface DiagramProps {
   animation?: DiagramAnimationType;
+  arrow?: {
+    fill?: boolean;
+    color?: ColorType;
+  };
   connections: {
     anchor?: DiagramConnectionAnchor;
     animation?: DiagramAnimationType;
     color?: ColorType;
+    decreaseLineLength?: Number;
+    elementPosition?: 'top' | 'bottom';
```

Comment:
I'm not sure what `elementPosition` drives. Is it which end of the line to put the arrowhead on? If so, I think we should tie this more to the targets. For example: `arrow: 'from'|'to'|true;`, with `true` meaning put an arrowhead on both ends.

Reply:
As mentioned above this prop is used differently, https://github.com/grommet/grommet/pull/5950#discussion_r831675683,
However, I haven't added the two way arrow thing, If you want I can add that too. Please, let me know.

Reply:
I would like to ensure that the API surface will allow for defining where the arrows are.

Reply:
I have added the `arrow: 'from'|'to'|true;` options